### PR TITLE
Removed static and created a prop for tabindex

### PR DESCRIPTION
**Pull request by JanRuneHenriksen**

### CODE REVIEW

**Comment by JanRuneHenriksen** [comment]
I also need help on how to do the rest of the "elementsui-react" process on how to get these changes into Elements.
- I guess that if you approve, I merge til to elementsui-react master
- And then you create an updated version of the npm package
- And then the package.json in Elements must be updated with the new version and pushed to Elements master

**Comment by JanRuneHenriksen** [comment]
@dmarcautan Review?

File: src/InfiniteList/InfiniteListItem.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import * as React from "react";
 import styled from "styled-components";
+import * as _ from "lodash";
```
Comment (dmarcautan) [inline src/InfiniteList/InfiniteListItem.tsx:3]: Seems like not needed anymore

Comment (JanRuneHenriksen) [reply src/InfiniteList/InfiniteListItem.tsx]: Fixed

**Review by dmarcautan — APPROVED** [review]

**Pull request merged**